ui/src/app/preferences/views/Details/DetailsButtons/DetailsButtons.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import ActionButton from "app/base/components/ActionButton";
+
+export const DetailsButtons = ({
```

Comment:
Just curious here, what makes this a `DetailsButtons` component? It seems like a generic component, maybe a `ButtonsPanel`?

Reply:
Just had another look and the other forms that have a button like this just include ActionButton directly, so I'll update this to do that.

Reply:
Oh actually now that I've looked at it again, the form needs to take a prop that is a reference to a component hence me having this small component for this one case.